subprojects/dependency-management/build.gradle.kts

```diff
@@ -63,6 +63,7 @@ dependencies {
     testImplementation(testFixtures(project(":base-services")))
     testImplementation(testFixtures(project(":snapshots")))
     testImplementation(testFixtures(project(":execution")))
+    testImplementation(testFixtures(project(":logging")))
```

Comment:
I don't think this is necessary anymore

Reply:
Right, it was only needed for the logging changes to `DefaultConfigurationSpec`, which are now removed.  Will remove.